lms/djangoapps/instructor/views/api.py

```diff
@@ -3772,7 +3774,8 @@ def _format_student_info_to_key_value(user_and_expiry_raw):
             else:
                 expiry = 'invalid'
 
-        if re.search(r'^[a-zA-Z]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', raw_user[0]):
+        raw_email = re.findall(r'[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+', raw_user[0])
```

Comment:
you are using this regex at multiple places, better to save its complied version at the top level.